### PR TITLE
Ms branch

### PR DESCRIPTION
**Pull request by communitybuildingcoders**
I have committed 4 times with each change, in the event you dislike the left alignment for the questions. I fixed the email input field from number type to email. Changed headings to uniformly match, dropped the semicolons, and aligned with boxes the way Personal Information was aligned. Fixed a few typos, rewrote the questions for clarity and professionalism. Added bullets and realigned questions to the left, helping to create more repetition and rhythm giving a more visually appealing document.
@ekelseya

### CODE REVIEW

File: form.html [hunk 1]
```diff
@@ -21,8 +21,9 @@ <h2 class="company">Coders Building Communities</h2><span><img class="logo" src=
             <h2 class="test-title hidden">No Test Selected</h2>
 
             <div id="slide-overview" class="questionnaire-slide slide-overview">
-                <p>Lorem ipsum, dolor sit amet consectetur adipisicing elit. Pariatur, maxime quis sunt et  <br>cumque nostrum tempore amet commodi
-                    possimus mollitia sit <br> minima impedit molestiae tempora tenetur fuga. Unde, quas numquam!</p>
+                <p>We are a group of programers and web developers looking to
+                    help our communities while building a better web one site at a time.
+                     <br> minima impedit molestiae tempora tenetur fuga. Unde, quas numquam!</p>
```
Comment (ekelseya) [inline form.html]: Instead of using a \<br\> in a \<p\> element, start a new \<p\>. It makes it easier for screen readers.

File: form.html [hunk 2]
```diff
@@ -21,8 +21,9 @@ <h2 class="company">Coders Building Communities</h2><span><img class="logo" src=
             <h2 class="test-title hidden">No Test Selected</h2>
```
Comment (ekelseya) [inline form.html]: Rather than having hidden elements, we should just remove this.

Comment (ekelseya) [inline form.html]: We need to add an alt tag to the logo (when we finalize it)

File: questionnaire.html [hunk 3]
```diff
@@ -8,105 +9,108 @@
...
-                <textarea id="comment1" placeholder="Enter your comment here..." rows="10"></textarea>
-            </span>
-        </fieldset>
-        <fieldset>
-            <span class="form-group">
-                <label id="comments-label">What’s the project timeline?</label>
-                <textarea id="comment2" placeholder="Enter your comment here..." rows="10"></textarea>
-            </span>
-        </fieldset>
+            <h1 id="title">CBCoders Website Questionnaire</h1><span><img class="logo"
```
Comment (ekelseya) [inline questionnaire.html:16]: Again, alt tag when we get a final logo

File: questionnaire.html [hunk 4]
```diff
@@ -8,105 +9,108 @@
...
+                    src="https://cdn.discordapp.com/attachments/754425697237467150/755120803166683387/cbcoders.png"
+                    alt="" srcset=""></span>
```
Comment (ekelseya) [inline questionnaire.html:18]: We should get rid of the empty srcset

File: questionnaire.html [hunk 5]
```diff
@@ -8,105 +9,108 @@
...
+            <p id="description">Please let us know how we can serve you.</p>
+            <hr>
+            <form id="survey-form">
+                <fieldset id="top">
+                    <legend>Personal Information</legend>
+                    <span class="form-group"><label id="name-label" for="name">Name: </label><input id="name"
+                            type="text" required placeholder="Enter you name"></span>
```
Comment (ekelseya) [inline questionnaire.html]: "Enter your name" (not you)

File: questionnaire.html [hunk 6]
```diff
@@ -8,105 +9,108 @@
...
+                                <li>What needs or goals do you expect from your website that aren’t being
+                                    met by the current version?</li>
+                                <li>What is the purpose of your website, what should your website achieve?</li>
+                                <li>What function does your website serve your brand?</li>
+                                <li>What do you expect visitors accomplish when they visit your website?</li>
+                                <li>Who is the website's intended target audience, who are you speaking to?</li>
+                                <li>What image, look, or feel do you want your brand’s website to portray?</li>
+                            </ul>
+                        </label>
+                        <textarea id="comment0" placeholder="Enter your comment here..." rows="10" cols="50"></textarea>
```
Comment (ekelseya) [inline questionnaire.html]: We do need to go back and fix the ids and labels for the textareas. Each one should have something descriptive.

File: form.css [hunk 7]
```diff
@@ -18,6 +18,11 @@ body {
     padding: 0;
```
Comment (ekelseya) [inline form.css:17]: I would increase font-size and line-height just a little bit. Makes everything pop and it's easier to read.

**Review by ekelseya — COMMENTED** [review]
I mostly crawled through this as an example of code review. We should be as detailed as possible when looking at code :) 

I love this.

File: form.html [hunk 8]
```diff
@@ -21,8 +21,9 @@ <h2 class="company">Coders Building Communities</h2><span><img class="logo" src=
             <h2 class="test-title hidden">No Test Selected</h2>
 
             <div id="slide-overview" class="questionnaire-slide slide-overview">
-                <p>Lorem ipsum, dolor sit amet consectetur adipisicing elit. Pariatur, maxime quis sunt et  <br>cumque nostrum tempore amet commodi
-                    possimus mollitia sit <br> minima impedit molestiae tempora tenetur fuga. Unde, quas numquam!</p>
+                <p>We are a group of programers and web developers dedicated to
+                    help our communities while building a better web one site at a time.
```
Comment (ekelseya) [inline form.html]: This falls off the darker background.

File: form.html [hunk 9]
```diff
@@ -34,8 +35,9 @@ <h3 class="how-it-works-h3">How it works</h3>
                     <dt>Let us do the rest. We are here to serve you!</dt>
                 </dl>
 
+
                 <div class="questionnaire-get-started">
-                  <button class="btn action-start">Let's get started</button>
+                  <button class="btn action-start" >Let's get started</button>
```
Comment (ekelseya) [inline form.html]: Normally I would bristle at a button as a link, but in this case, I think it makes sense. A button does something, and this one starts the questionnaire.

File: form.html (hunk 1 above)
Comment (communitybuildingcoders) [reply form.html]: The <br> was there already I left it in the event someone else had something planned for that area :)

**Review by chrisowensdev — COMMENTED** [review]
I like the change from comment to response

File: form.html (hunk 1 above)
Comment (ekelseya) [reply form.html]: GitHub treats HTML tags as *HTML tags* so you can make fancy comments. To escape the formatting, add a backslash \\ to the angle bracket.

\\<p\\> becomes \<p\>

**Comment by communitybuildingcoders** [comment]
Deleting branch as likely outdated changes included.

**Pull request closed without merging**